WordPress/src/main/res/values/strings.xml

```diff
@@ -986,6 +986,30 @@
     <string name="editor_post_settings_set_featured_image">Set Featured Image</string>
 
 
+    <!-- Aztec -->
+    <string name="dialog_title">Insert link</string>
+    <string name="dialog_edit_hint">http(s)://</string>
```

Comment:
This one's untranslatable - we should either mark it as such or leave it out of the main project's `strings.xml` (but mark it untranslatable in the editor repo anyway).